Description: remove wasm32 architecture from Cargo.toml
Author: Nadzeya Hutsko <nadzya.info@gmail.com>
Forwarded: not-needed
Last-Update: 2025-08-23
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -91,35 +91,10 @@
     "bitflags/serde",
 ]
 std = [
-    "js-sys?/std",
-    "web-sys?/std",
     "thiserror/std",
 ]
 strict_asserts = []
 trace = ["std"]
-web = [
-    "dep:js-sys",
-    "dep:web-sys",
-]
-
-[target.'cfg(target_arch = "wasm32")'.dependencies.js-sys]
-version = "0.3.77"
-optional = true
-default-features = false
-
-[target.'cfg(target_arch = "wasm32")'.dependencies.web-sys]
-version = "0.3.77"
-features = [
-    "ImageBitmap",
-    "ImageData",
-    "HtmlImageElement",
-    "HtmlVideoElement",
-    "HtmlCanvasElement",
-    "OffscreenCanvas",
-    "VideoFrame",
-]
-optional = true
-default-features = false
 
 [lints.clippy]
 alloc_instead_of_core = "warn"
@@ -129,4 +104,3 @@
 [lints.rust.unexpected_cfgs]
 level = "warn"
 priority = 0
-check-cfg = ["cfg(web_sys_unstable_apis)"]
